styleguide/components/31-molecules/menu/_menu.scss

```diff
@@ -12,8 +10,8 @@ nav.menu {
   .drawer {
     position: fixed;
     top: 0;
-    left: -$hamburger-menu-width - 16px;
-    width: $hamburger-menu-width;
+    left: calc(-100% - 16px);
```

Comment:
Why pixels instead of rems here?

Reply:
because the box-shadow is also in px :)

Reply:
Oh... shouldn't we change the box shadow to rems then? ;)

Reply:
This is okay for me then :)